sdk/python/pulumi_kubernetes/admissionregistration/v1/MutatingWebhookConfiguration.py

```diff
@@ -1,55 +1,41 @@
-# *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+# coding=utf-8
+# *** WARNING: this file was generated by pulumigen. ***
 # *** Do not edit by hand unless you're certain you know what you are doing! ***
 
+import json
```

Comment:
Any idea why this is included? Doesn't look like it's used in this file.

Reply:
Nope, you're not missing anything. Can you file an issue to track adding this conditionally?

Reply:
I created an issue https://github.com/pulumi/pulumi/issues/4890
It's not entirely straightforward adding this conditionality so I'll let the issue go through prioritizing.